docs/agent-api.asciidoc

```diff
@@ -477,6 +480,20 @@ if used within the context of a transaction.
 NOTE: If there's no active transaction available,
 `null` will be returned.
 
+[[apm-current-span]]
+==== `apm.currentSpan`
+
+[[source,js]]
```

Comment:
```suggestion
[source,js]
```

Reply:
Also update line 472 to `[source,js]`

Reply:
@watson ^

Reply:
Thanks. I'll update line 472 in a separate PR as I saw that this isn't the only place this mistake snug in 😅